packages/terra-clinical-app-delegate/docs/README.md

```diff
@@ -0,0 +1,138 @@
+# Terra Clinical App Delegate
+
+The AppDelegate is an object that provides a set of functions used to facilitate communcation between Containers and
+their consuming Containers/Applications.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-clinical-app-delegate`
+
+## Usage
+
+```js
+import AppDelegate from 'terra-clinical-app-delegate';
+
+const newAppDelegateInstance = AppDelegate.create({
+  disclose: (data) => {
+    // Perform disclose action (open modal, inline panel, etc.)
+  },
+  dismiss: (data) => {
+    // Dismiss current Container
+  },
+  closeDisclosure: (data) => {
+    // Close entire disclosure method, regardless of other content
+  },
+  goBack: (data) => {
+    // Dismiss current Container
+  },
+  maximize: (data) => {
+    // Maximize the disclosure method
+  },
+  minimize: (data) => {
+    // Minimize the disclosure method
+  }
+});
+
+const clonedAppDelegateInstance = AppDelegate.clone(newAppDelegateInstance, {
+  disclose: (data) => {
+    // Override disclose function from newAppDelegateInstance
+  },
+});
+
+```
+
+
+The implementations of the AppDelegate APIs are completely determined by the component creating the AppDelegate instance. However,
+the suggested use of each function is as follows:
+
+|Name|Status|Description|
+|---|---|---|
+|`disclose`|Required|Used to progressively disclose content in various ways (modal, inline, etc.)|
+|`dismiss`|Optional|Used to dismiss the calling component. Should only be present if the component is being presented in a closable disclosure.|
```

Comment:
`Should only be present if the component is being presented in a closable disclosure.`
dismissible disclosure

You should define the nuanced difference between close and dismiss.